### ISSUE TITLE
MOL content info & taxon selection dashboard

### ISSUE THREAD

**Issue by walterj** [MapofLife/MOL#92]
At least in simple(st) form needed for beta release. Below just an outline of what this could look like (not a request, just a suggestion for discussion).
# Looks etc:

E.g. Size: ca. ½ to ¾ of screen in front of map view (which is greyed out, behind).
Color: black or grey with whitish fonts (or whatever is preferred)?
Appears/disappears: appears by click on dashboard button (top left of screen), disappears by  closing button or click on map behind)
# INFO appearance:

Tab 1: by Taxon, simple taxonomic list or, better, (tree) browser that vertically lists every ~ Class (e.g. “Mammals”, “Birds”, etc.), and, by e.g. clicking on + button, Order. For every row (e.g. “Birds” or “Birds” – “Hummingbirds”) and for each data type (for now: points, expert maps, checklists) the total number of species with data, total number of records (i.e. points or different expert maps [e.g. IUCN vs. Jetz], study area occurrences) and median number of records per species is given. So for now this would result in 3 x 3 cells. All statistics should be calculated on the fly or pre-calculated weekly.
View 2: by data type. For now only three: points, expert maps, checklists. For each one can then see for each Class (e.g. “Mammals”) the same statistics as above.
View 3: by data provider. Outer: IUCN, GBIF, WWF, XXX Lab, YYY Lab. For each one can then see for each Class (e.g. “Mammals”) the same statistics as above. All provider names should be clickable for metadata, links.
# LIST tool appearance

When clicked, user is taken to a slightly different version of same dashboard, i.e. the simply taxonomy (Class, Order) browser. But now only the holdings available under species list tool are shown (right now: expert map holdings). User can select single Class (or single/multiple Orders within a single Class, not urgent)  via  box. After pushing ‘OK’ user is taken to same map window as in 1) (except that the ‘search’ button for mapping single species is e.g. greyed out?). Selected taxa for list query show up e.g. bottom left. User can move pointer with circle size of ca. 100km and click do get ‘point’ list produced, or with same pointer can draw polygon to produce ‘area’ list (behind the scenes, that polygon would need to be buffered out by ca. 100km to address expert map boundary uncertainty). Then user gets in window a structured response that has as title the Class/Order(s) selected, the area (including buffer) queried, the count of species, and then, in taxonomic structure/order, the species in English. [After demo release: . Upon pushing a button, users can select one to several additional columns for output: Class, Order, Latin, taxonomy used etc.. These can be displayed or downloaded as csv. User can change taxon selection by clicking on taxon list (bottom left), gets taken back to taxon selection window.]

How does this sound?

**Comment by eightysteele** [comment]
Yeah man, this sounds awesome! Unfortunately it's not a quick thing to develop given our capacity and load. We're 10 days from release and I think we need a feature freeze given our 27 open issues. I think we should table this until end-of-month, and then definitely give it some love for the next development phase.

**Comment by robgur** [comment]
Hey all. I agree we should motor on open issues to end of month, but would like to put a _very simple_ start of the dashboard on the table for March 5 --- I think at least initially we could have a little button on the interface that pulls up something that looks like the individual feature metadata interface that shows datasets we have ingested and the number of records per higher level taxon (Birds, Mammals, Amphibians and Reptiles).  Doable?

**Comment by eightysteele** [comment]
Maybe. Let's firm up what it means to "show datasets we have ingested and the number of records per higher level taxon". We could probably surface something like this pretty quickly in the UI:

```
GBIF points: 300 Birds, 500 Mammals, 200 Amphibians, 100 Reptiles
MOL range maps: 10,000 Birds, 2000 Mammals, 5000 Amphibians, 5000 Reptiles
IUCN range maps: 4,000 Birds, 1000 Mammals, 8000 Amphibians, 2000 Reptiles
...
```

I imagine a "data" link that simply shows the above summary. Thoughts?

**Comment by walterj** [comment]
Yes, something like this in  the very simplest form, I guess. But I would say let's give it a tabular look, i.e. with taxa as columns and data types as rows? For points and checklists etc we need to distinguish # records and # species, but this could be done with additional columns (or rows). Should all not be too hard hopefully?

**Comment by eightysteele** [comment]
Yeah, tabular form for sure that includes record count and unique species count. Maybe something like this:

<table>
<tr>
<td><b>Source</b></td>
<td><b>Amphibians</b></td>
<td><b>Birds</b></td>
<td><b>Mammals</b></td>
<td><b>Reptiles</b></td>
</tr>
<td>GBIF points</td>
<td>500 species with 23,000 records</td>
<td>1,500 species with 30,000 records</td>
<td>2,000 species with 20,000 records</td>
<td>800 species with 100,000 records</td>
</tr>
</table>

**Comment by walterj** [comment]
Thanks, Aaron. Yeah, for the beginning this (with additional rows etc. ) may do.

Walter

**Comment by eightysteele** [comment]
Yeah, this is a reasonable starting point. We'll have a row per source/type for sure. Adding this issue to release milestone with priority.
